### PR TITLE
Con 937 add paye scheme

### PR DESCRIPTION
**Pull request by Stephek**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -44,6 +44,7 @@
     <!--[if IE 8]><link href="@Styles.Url("~/bundles/screenie8")" media="screen" rel="stylesheet" /><![endif]-->
     <!--[if gt IE 8]><!-->
     <link href="@Styles.Url("~/bundles/screen")" media="all" rel="stylesheet" />
+    <link href="@Html.CdnLink("css", "main.css")" rel="stylesheet" />
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml]: We don't want to pull the styles from the CDN, work is being done to copy the styles required into the local stylesheet for now.

**Comment by Stephek** [comment]
> 1 change regarding pulling the cdn into the V1 layout. Only other thing is, looking at the code, moving everything out of the EmployerTeam Index.cshtml makes for a little more confusing rendering the Rows and Panels which aren't so much a thing in this design. Could we have just called the ChildAction for rendering the call to action panel at the top of the existing view?. Just a thought, not saying we revert it all back

**Comment by Stephek** [comment]
I'm assuming that the page will morph towards the layout of the new homepage so separated the layout from the content to more easily allow for this going forward.

File: src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (Stephek) [reply src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml]: Should I remove?  This story will be blocked until that work is complete.

**Comment by cofaulco** [comment]
Feature toggled call to action for the 4 panel homepage which is no longer applicable. Closing PR and deleting branch

**Pull request closed without merging**